media/css/firefox/firstrun/firstrun-quantum.scss

```diff
@@ -32,136 +31,59 @@ $mq-large:  'screen and (min-width: 860px)';
     --animation-curve: linear;
 }
 
+html {
+    height: 100%;
+}
+
 body {
+    background: #003eaa;
+    background: #003eaa url("/media/img/firefox/firstrun/fox-tail-header.png") top -200px center no-repeat;
+    height: 100%;
     margin: 0;
-    overflow: hidden;
-
-    @media (max-height: 710px) {
-        overflow-y: auto;
+    overflow-y: auto;
+    @supports (display: grid) {
```

Comment:
Not a problem to leave the `@supports` checks in, but since this page is only being served to Fx versions that support grid... 😉